### PR TITLE
move eraftpb out of kvproto

### PR DESCRIPTION
**Pull request by BusyJay**
This pr move eraftpb from kvproto into this crate. So that we can drop the dependency of kvproto and grpc.

/cc #1

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @Hoverbear

File: proto/eraftpb.proto [hunk 1]
```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+package eraftpb;
+
+option java_package = "com.pingcap.tikv.eraftpb";
```
Comment (overvenus) [inline proto/eraftpb.proto]: Do we still need it?

Comment (BusyJay) [reply proto/eraftpb.proto]: Yes. Otherwise I don't think TiSpark can use kvproto anymore.

Comment (overvenus) [reply proto/eraftpb.proto]: I haven't found any reference to eraftpb in TiSpack.
https://github.com/pingcap/tispark/search?utf8=%E2%9C%93&q=eraftpb&type=

Comment (BusyJay) [reply proto/eraftpb.proto]: It's an indirect dependency. It depends on pdpb, which depends on eraftpb.

**Comment by siddontang** [comment]
ping @overvenus

File: proto/eraftpb.proto (hunk 1 above)
Comment (overvenus) [reply proto/eraftpb.proto]: Any suggestions? @ilovesoup

**Comment by BusyJay** [comment]
Ping

File: proto/eraftpb.proto (hunk 1 above)
Comment (siddontang) [reply proto/eraftpb.proto]: ping @ilovesoup

Comment (ilovesoup) [reply proto/eraftpb.proto]: Just did a test seems fine for TiSpark. Please feel free go on.

Comment (siddontang) [reply proto/eraftpb.proto]: @BusyJay 

We can remove this now.

**Comment by overvenus** [comment]
LGTM

**Pull request merged**